### PR TITLE
Release some resources

### PR DESCRIPTION
**Pull request by CaiJingLong**
1. recycle bitmap
2. close stream

### CODE REVIEW

File: android/src/main/java/com/example/flutternativeimage/FlutterNativeImagePlugin.java [hunk 1]
```diff
@@ -149,16 +145,18 @@ public void onMethodCall(MethodCall call, Result result) {
       }
 
       bmp.compress(Bitmap.CompressFormat.JPEG, 100, bos);
-
+      bmp.recycle(); // recycle bitmap
```
Comment (CaiJingLong) [inline android/src/main/java/com/example/flutternativeimage/FlutterNativeImagePlugin.java]: recycle bitmap

File: android/src/main/java/com/example/flutternativeimage/FlutterNativeImagePlugin.java [hunk 2]
```diff
@@ -167,9 +165,19 @@ public void onMethodCall(MethodCall call, Result result) {
...
   		} catch (IOException e) {
-  			e.printStackTrace();
-        result.error("something went wrong", fileName, null);
-  		}
+            e.printStackTrace();
+            result.error("something went wrong", fileName, null);
+        }finally {
+            if (outputStream != null) {
+                try {
+                    outputStream.close(); // close file OutputStream
```
Comment (CaiJingLong) [inline android/src/main/java/com/example/flutternativeimage/FlutterNativeImagePlugin.java]: The stream is FileOutputStream, need close.

**Comment by CaiJingLong** [comment]
In addition, the first idea I had when I saw code was to format it.

**Comment by btastic** [comment]
Looks fine, can you have a look over the comments I made?

**Comment by CaiJingLong** [comment]
Sorry, I didn't see your comment. Where is it?

**Comment by btastic** [comment]
Can you see it now?

File: android/src/main/java/com/example/flutternativeimage/FlutterNativeImagePlugin.java [hunk 3]
```diff
@@ -213,7 +221,7 @@ private void copyExif(String filePathOri, String filePathDest) {
       newExif.saveAttributes();
 
     } catch (Exception ex) {
-      Log.e("FlutterNativeImagePlugin", "Error preserving Exif data on selected image: " + ex);
+      Log.e("FlutterNativeImagePlugi", "Error preserving Exif data on selected image: " + ex);
```
[the remark this answers is not in the document]
Comment (btastic) [reply android/src/main/java/com/example/flutternativeimage/FlutterNativeImagePlugin.java]: You removed a letter in the Log.e("FlutterNativeImagePlugi**n**", ..)

**Comment by CaiJingLong** [comment]
I removed comment. 
I'll explain TAG's problem, if you don't like the reason, I can modify it.

**Comment by btastic** [comment]
Feel free to explain, but it seems unnecessary..

**Comment by CaiJingLong** [comment]
Do you need me to change the TAG back?

**Comment by btastic** [comment]
> Do you need me to change the TAG back?

Well if you have a good reason, it can stay. But I don't see why.

**Comment by CaiJingLong** [comment]
There are some discussions in [Stack Overflow](https://stackoverflow.com/questions/10514175/which-android-logging-framework-to-use) about the android log framework, which mentions the limitations of the Android platform on TAG.
It doesn't matter. I'll change it back.

**Comment by btastic** [comment]
> There are some discussions in [Stack Overflow](https://stackoverflow.com/questions/10514175/which-android-logging-framework-to-use) about the android log framework, which mentions the limitations of the Android platform on TAG.
> It doesn't matter. I'll change it back.

I did not find the reason on your stack overflow link. However, I'd still say that the TAG change should be in a different PR.

**Comment by btastic** [comment]
Thank you very much!

**Comment by CaiJingLong** [comment]
> > There are some discussions in [Stack Overflow](https://stackoverflow.com/questions/10514175/which-android-logging-framework-to-use) about the android log framework, which mentions the limitations of the Android platform on TAG.
> > It doesn't matter. I'll change it back.
> 
> I did not find the reason on your stack overflow link. However, I'd still say that the TAG change should be in a different PR.

Another PR is right.

**Pull request merged**